task-maker-async/src/store.rs

```diff
@@ -35,9 +35,11 @@ enum HandleMode {
     Write,
 }
 
-#[derive(Debug, Serialize, Deserialize)]
+type HandleId = usize;
+
+#[derive(Debug, Serialize, Deserialize, Clone, Copy)]
 pub struct FileSetHandle {
-    id: usize,
+    id: HandleId,
```

Comment:
Are the ids of a `FileSetHandle` and of a `FileHandle` interchangeable? If not, I'd prefer `FileSetHandleId` and `FileHandleId`. This makes 100% clean what the hashmap are indexed with.